apps/extension/src/ui/domains/Asset/Fiat.tsx

```diff
@@ -74,11 +52,62 @@ export const Fiat = ({
           amount={isHidden ? 0 : typeof amount === "number" ? amount : amount.fiat(currency) ?? 0}
           currency={currency}
           currencyDisplay={currencyDisplay}
+          isBalance={isBalance}
           noCountUp={effectiveNoCountUp}
         />
       )}
     </span>
   )
 }
 
-export default Fiat
+const DisplayValue = ({
+  amount,
+  currency,
+  currencyDisplay,
+  isBalance,
+  noCountUp,
+}: DisplayValueProps) => {
+  /**
+   * Represents the first non-zero decimal place for numbers between -1 and 1
+   * For example:
+   *
+   *     Input  - Output
+   *     ****** - ******
```

Comment:
This is a great comment, but we could improve it by making clear what the actual output is - eg `99` instead of `0` for input `99.000`

Reply:
Hmmmmm, the output of `99.000` should be `0`. Are you certain that you're getting `99` from it?

Reply:
```shell
> Math.abs(Math.min(0, Math.floor(Math.log10(Math.abs(99.000)))))
0
```